### ISSUE TITLE
[Feature Request] Disable WebUI Authentication

### ISSUE THREAD

**Issue by jwgn** [ThomasVon2021/blikvm#232]
**Problem Description**
As I currently use my BLIKVM v4 only in my home network, it would be convenient not having to enter credentials for WebUI access.

**Proposed Solution**
Add an option to disable WebUI authentication entirely

**Alternatives Considered**
None.

**Additional context**
None.

**Comment by m50S79sM6SRNp8Jn** [comment]
Do you by any chance use a Yubikey?

**Comment by jwgn** [comment]
I use multiple hardware keys, but I do not intend to move them between my devices just to access a WebUI. It would be more complicated than entering the login credentials.

**Comment by m50S79sM6SRNp8Jn** [comment]
Would you use a password manager? Would that allow you to automatically login, while using authentication credentials?

**Comment by jwgn** [comment]
This is not what I asked for. My request was to disable WebUI authentication entirely.

**Comment by m50S79sM6SRNp8Jn** [comment]
True. Was just thinking of an other / near immediate way of dealing with the inconvenience of having to enter credentials for WebUI access.

**Comment by ThomasVon2021** [comment]
you can modify /mnt/exec/release/app.json 
find auth, set to false

**Comment by jwgn** [comment]
Thanks, it works!

Correct path is `/mnt/exec/release/config/app.json` :)
